### DISCUSSION TITLE
Using "Upscale 2x" from the "Enhance" tab is always using the first generated prompt when using wildcards. Any fix?

### DISCUSSION THREAD

**Discussion by Jetpack-art** [lllyasviel/Fooocus#3563] [Q&A]
Hello everyone,

I've been using Fooocus for a while now and the "Enhance" process that automate the "Upscale 2x" when generating hundred of pics has been a huge help.

However, I'm using several wildcards in my prompt and I noticed that the "Enhance" process always uses the first prompt generated out of the wildcards. If I generate 100 pics, this results in a problem that the Upscale 2x process sometimes affect the faces or people in all the pics making them look like the very first generated prompt.

To better explain with an example, I have a wildcard that I use for facial expressions. For example, it can be:

- smiling
- annoyed
- crying

If I generate 100 pics with the wildcard and that the first selected prompt out of that card is "crying", all the initially generated pics will be Upscaled 2x but they will end up with the crying faces on all of them (whereas the initial generated pics of the first pass are different and using the right randomly generated prompt.)

Is there a way to make sure that the "Enhance Upscale 2x" process use the associated prompt of each pic?

Thank you for your help!

**Comment by Jetpack-art** [comment]
Up! :)

**Reply by Yogesh-DevHub** [reply]
Well, there is a fork I was looking at for bulk image processing a week ago, but my client later denied the job, and I did not attempt. 
https://github.com/ChrisColeTech/Fooocus
